cheroot/test/helper.py

```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```

Comment:
When doing pull requests you may update it by pushing new commits into your branch or rewrite it with rebase and force push. There's no need to create new PR after each review :) 
Moreover, if you update commented lines, the related comments will automatically become collapsed